src/lib/Microsoft.Health.Fhir.Ingest.Template/InvalidTemplateException.cs

```diff
@@ -39,5 +41,9 @@ public override string Message
         public ILineInfo GetLineInfo => _lineInfo;
 
         public bool HasLineInfo => _lineInfo.HasLineInfo();
+
+        public override string ErrSource => nameof(ErrorSource.User);
+
+        public override string ErrSeverity => ErrorSeverity.Critical;

```

Comment:
is the error severity exposed to the customer?

wondering if it would be useful to have Critical and Warning for errors as well as "Information" traces in the future.

Reply:
Currently, it's not. I think I can expose this and make a default severity level for unstructured exceptions. Do you think I should add this to the current PR or should it be a part of the next iteration?

Reply:
I believe it is exposed on the error metrics we emit today.